js/pro/binance.js

```diff
@@ -71,7 +71,20 @@ module.exports = class binance extends binanceRest {
                 'wallet': 'wb', // wb = wallet balance, cw = cross balance
                 'listenKeyRefreshRate': 1200000, // 20 mins
                 'ws': {
-                    'cost': 5,
+                    'rateLimits': {
+                        'default': {
+                            'tokenBucket': {
```

Comment:
we don't need this anymore if we intend to infer the value from the rate limit right?